tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -110,10 +132,15 @@ Spawn Notebook With Arguments
             Add Spawner Environment Variable  ${key}  ${value}
          END
       END
-      Click Button  Start server
-      Wait Until Page Contains  Your server is starting up
+      ${version-check} =  Is RHODS Version Greater Or Equal Than  1.5.0
+      IF  ${version-check}==True
+         Click Button  Start Server
```

Comment:
I think you can move this line of code outside the IF statement because duplicate